unreal/MicromegasTelemetrySink/MicromegasTelemetrySink.Build.cs

```diff
@@ -12,7 +12,9 @@ public MicromegasTelemetrySink(ReadOnlyTargetRules Target) : base(Target)
 			"Core",
 			"CoreUObject",
 			"HTTP",
+			"BuildSettings",
 			"Engine",
+			"RHI",
```

Comment:
We can discuss if this is too much of a dependency to drag on. It's already pulled by the server target from what I can see thought.